### DISCUSSION TITLE
Release-v1.2.0

### DISCUSSION THREAD

**Discussion by pankajcs033** [vitabletech/SnippetMaster#27] [Announcements]
## What's Changed
* enhancement: new enhancement add new git cmd for reset author details by @msrajawat298 in https://github.com/vitabletech/SnippetMaster/pull/19
* merge Develop branch with release branch bcz package realese by @msrajawat298 in https://github.com/vitabletech/SnippetMaster/pull/21
* after release merge into main Release v1.1.1 by @msrajawat298 in https://github.com/vitabletech/SnippetMaster/pull/23
* after release by @msrajawat298 in https://github.com/vitabletech/SnippetMaster/pull/24
* New Feature added related to  Wordpress code snippet by @pankajcs033 in https://github.com/vitabletech/SnippetMaster/pull/25
* New Feature added related to  Shell code snippet by @pankajcs033 in https://github.com/vitabletech/SnippetMaster/pull/25
* Merge changes into release branch by @pankajcs033 in https://github.com/vitabletech/SnippetMaster/pull/26

## New Contributors
* @pankajcs033 made their first contribution in https://github.com/vitabletech/SnippetMaster/pull/25

**Full Changelog**: https://github.com/vitabletech/SnippetMaster/compare/v1.1.0...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/vitabletech/SnippetMaster/releases/tag/v1.2.0'>Release-v1.2.0</a>.</em>

**Comment by msrajawat298** [comment]
hurray
